ansible/pbTestScripts/testScript.sh

```diff
@@ -188,8 +190,13 @@ startVMPlaybookWin()
 		# Add the "ansible_winrm_transport" to adoptopenjdk_variables.yml
 		echo -e "\nansible_winrm_transport: credssp" >> playbooks/AdoptOpenJDK_Windows_Playbook/group_vars/all/adoptopenjdk_variables.yml
 	fi
-        # Resize the c drive, or return true if it's already been resized
-        vagrant powershell -c "Start-Process powershell -Verb runAs; \$size = (Get-PartitionSupportedSize -DriveLetter c); Resize-Partition -DriveLetter c -Size \$size.SizeMax" || true
+	# getting the current c drive information and cutting it down to the number of GB it is.
+	export currentDiskSize=$(vagrant powershell -c "Start-Process powershell -Verb runAs; Get-Partition -Driveletter c" | grep "GB")
+	currentDiskSize=$(echo $currentDiskSize | cut -d ' ' -f 5 | cut -d '.' -f 1)
+	if [[ $currentDiskSize -lt $diskSizeBoundary ]]; then
```

Comment:
My only concern with doing this is that if the units are in TB then this could get rather confused - can you run `vargrant powershell` in a way that allows you to run the `select` statement afterwards to show the raw number without units?

Reply:
Fixed :) It's now done in bytes